gateway-js/src/make-fetch-happen.d.ts

```diff
@@ -33,6 +33,10 @@ declare module 'make-fetch-happen' {
           randomize?: boolean;
         };
     onRetry?(): void;
+    cert?: string;
+    key?: string;
```

Comment:
```suggestion
    cert?: SecureContextOptions['cert'];
    key?: SecureContextOptions['key'];
```
We'll also need to add the proper import at the top:
`import { SecureContextOptions } from 'tls';`

Reply:
Ref: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/tls.d.ts#L563